compose.yml

```diff
@@ -2,7 +2,7 @@ version: "2.0"
 
 services:
   mysql:
-    image: mysql:8.0
+    image: mysql:5.7
```

Comment:
do you want to revert this?

Reply:
I think so, just for simplicity sake so that the tagged 2.6.3 version remains completely unchanged. It doesn't have any real effect anyways.